<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>structure_Rotation_Vec</class>
 <widget class="QDialog" name="structure_Rotation_Vec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>向量旋转</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入旋转向量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Structure_Rotation_Vec_Vector_y">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Structure_Rotation_Vec_Speed">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Structure_Rotation_Vec_Start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Structure_Rotation_Vec_Vector_x">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Structure_Rotation_Vec_End">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Structure_Rotation_Vec_Vector_z">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
